Sources/NIO/ChannelOption.swift

```diff
@@ -190,6 +190,12 @@ extension ChannelOptions {
 
             public init() { }
         }
+        
+        /// When set to true IP level ECN information will be reported through `AddressedEnvelope.MetaData`
```

Comment:
World's nitpickiest nit!

```suggestion
        /// When set to true IP level ECN information will be reported through `AddressedEnvelope.Metadata`
```